dev-packages/node-integration-tests/suites/express/tracing/withError/server.js

```diff
@@ -0,0 +1,29 @@
+const { loggingTransport } = require('@sentry-internal/node-integration-tests');
```

Comment:
Side note: This is not the only integration test that covers `event.transaction` values in Express. There are a bunch more but they didn't change because changes 1 and 3 (see description) cancel each other out. Which IMO is good news :D